server/conf/cloudstack-catalina7.logrotate

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+/var/log/cloudstack/management/catalina.out {
+    copytruncate
+    daily
+    rotate 14
+    compress
+    missingok
+    create 0644 cloud cloud
+    su cloud cloud
```

Comment:
The only difference is 'su cloud cloud', will it break if we include this change in the tomcat6 based logrotate file?

Reply:
any update @rhtyd